### DISCUSSION TITLE
ByteBuffer and the Dreaded NoSuchMethodError - Gunnar Morling

### DISCUSSION THREAD

**Discussion by giscus** [gunnarmorling/discussions.morling.dev#294] [Announcements]
# ByteBuffer and the Dreaded NoSuchMethodError - Gunnar Morling

The other day, a user in the Debezium community reported an interesting issue;
They were using Debezium with Java 1.8 and got an odd NoSuchMethodError:

https://www.morling.dev/blog/bytebuffer-and-the-dreaded-nosuchmethoderror/

**Comment by autumoswitzerland** [comment]
Perfect, thanks!

**Comment by jtnystrom** [comment]
Thank you for writing this up. Saved me a lot of trouble.

**Comment by amilarajans** [comment]
Same here, saved me a lot of time

**Comment by ilan** [comment]
Someone pointed me to your link and it perfectly fit my problem.
I'm writing software for Fiji https://fiji.sc/
where I ran into the problem your described so nicely.
I would just like you to know that it helped me over a difficult problem.
